Point start
Point end
Point center
float64 radius
float64 arc_start
float64 delta
bool direction
bool orientation
string configIntervalType  # TCI: szakasz, ACI: körív
